Tests/js/apps/assets/ez-platformuiapp-tests.js

```diff
@@ -1144,6 +1144,41 @@ YUI.add('ez-platformuiapp-tests', function (Y) {
             Y.Assert.isFalse(removed, "The side view should not be removed");
             Y.Assert.isTrue(nextCalled, "The next callback should have been called");
         },
+
+        "Should destroy the sideViews": function () {
+            var sideViewInstanceMock = new Y.Mock(),
+                sideViewServiceMock = new Y.Mock();
+
+            Y.Mock.expect(sideViewInstanceMock, {
+                method: 'destroy',
+                args: [Y.Mock.Value.Object],
```

Comment:
that's not enough, you should add a `run` function and check that the object in parameter has a `property` remove set to `true`.